### PR TITLE
DEV-1315: fabs easy delete records

### PR DESCRIPTION
**Pull request by alburde1**
**High level description:**
Allowing users to submit only the bare minimum for FABS delete records

**Technical details:**
Updating almost all rules to ignore when correction delete indicator is D. Removing requirements for all columns except CFDA Number. Only deriving mod date and a couple other key details when publishing deletes.

**Link to JIRA Ticket:**
[DEV-1315](https://federal-spending-transparency.atlassian.net/browse/DEV-1315)

The following are ALL required for the PR to be merged:
- [ ] Backend review completed
- [x] Unit & integration tests updated with relevant test cases
- Frontend impact assessment completed

### CODE REVIEW

File: tests/unit/dataactvalidator/test_fabsreq1_detached_award_financial_assistance.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    expected_subset = {'row_number', 'award_description', 'correction_delete_indicatr'}
+    actual = set(query_columns(_FILE, database))
+    assert expected_subset == actual
+
+
+def test_success(database):
+    """ Test AwardDescription is required for all submissions except delete records. """
+
+    det_award = DetachedAwardFinancialAssistanceFactory(correction_delete_indicatr='C', award_description='Yes')
+    det_award_2 = DetachedAwardFinancialAssistanceFactory(correction_delete_indicatr='', award_description='No')
```
Comment (dpb-bah) [inline tests/unit/dataactvalidator/test_fabsreq1_detached_award_financial_assistance.py]: Can we include a passing test where correction is `D` and award description is not None or ''? (applies to all the test_fabsreq passing tests)
